src/components/MultiHopTrade/hooks/useReceiveAddress.tsx

```diff
@@ -72,10 +75,11 @@ export const useReceiveAddress = () => {
         wallet,
         accountMetadata: buyAccountMetadata,
         deviceId: await wallet.getDeviceID(),
+        ...(fetchUnchainedAddress && { pubKey: fromAccountId(buyAccountId).account }),
```

Comment:
nitpick but im a fan of avoid the use of object_spread for populating a single key
```suggestion
        pubKey: fetchUnchainedAddress ? fromAccountId(buyAccountId).account : undefined,
```

Reply:
Might as well, as well! 9293cf6c86